samples/adaptive_load/example_session_spec.textproto

```diff
@@ -0,0 +1,109 @@
+# proto-file: adaptive_load/adaptive_load.proto
```

Comment:
What is our long-term plan with this example?

Reply:
Deleting this for now. The codelab to be developed later can probably do a better job of illustrating the options, and may or may not have a checked-in sample file.